### ISSUE TITLE
Android emoticons

### ISSUE THREAD

**Issue by Fab2100** [kontalk/androidclient#461]
Android emoticons as an alternative to the ios ones would be great.

**Comment by daniele-athome** [comment]
Emojicons already supports them by simply setting a flag (but only for Android 4.2.2+ where emojis are builtin into the system).

**Comment by Fab2100** [comment]
How can I set a flag?

**Comment by daniele-athome** [comment]
I'm sorry that was a note for me :-) the library we use for the emojis allow to switch from Apple emojis to Android emojis easily, but only for Android 4.2+

**Comment by breversa** [comment]
I guess it must be set at compile time, right ?

**Comment by daniele-athome** [comment]
No, the library we use does allow switching at runtime, however a new preference is needed to let the user switch.

**Comment by LazerL0rd** [comment]
Could you add the possibility to use EmojiOne too?

**Comment by daniele-athome** [comment]
Great collection, thanks, I'll look into it.
